src/helper/selection-tools/reshape-tool.js

```diff
@@ -232,6 +232,11 @@ class ReshapeTool extends paper.Tool {
         this.active = false;
     }
     handleKeyDown (event) {
+        if (event.event.target instanceof HTMLInputElement) {
+            // Ignore nudge if a text input field is focused
+            return;
+        }
```

Comment:
Is this here intentionally? Looks unrelated. Probably fine to leave it though.

Reply:
This is vaguely related. Currently, if you are on the reshape tool and you have something selected, and you also have a textbox focused, if you press the arrow keys, both the selected thing and the cursor position will move. This return makes it so that only the cursor position moves. I will update the description.